db/migrate/20210520172440_devices_cookie_uuid_index.rb

```diff
@@ -0,0 +1,7 @@
+class DevicesCookieUuidIndex < ActiveRecord::Migration[6.1]
+  disable_ddl_transaction!
+
+  def change
+    add_index :devices, :cookie_uuid, algorithm: :concurrently
+  end
```

Comment:
this is a big migration! do we want to cherry-pick it to its own deploy?

Reply:
yes we will run this migration ahead of recycle